src/mds/MDSRank.cc

```diff
@@ -2923,7 +2933,7 @@ void MDSRankDispatcher::evict_clients(
     Session *s = p.second;
 
     if (filter.match(*s, std::bind(&Server::waiting_for_reconnect, server,
-				   std::placeholders::_1))) {
+                                   std::placeholders::_1))) {
```

Comment:
Unnecessary whitespace noise here and below.

Reply:
Removed